src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.ts

```diff
@@ -189,27 +162,29 @@ export class GeneNominatedTargetsComponent implements OnInit {
     return self.indexOf(value) === index;
   }
 
-  commaFlattenArray(array: any[]): any[] {
-    const finalArray: any[] = [];
-    if (array.length) {
-      array.forEach((t) => {
-        if (t) {
-          const i = t.indexOf(', ');
-          if (i > -1) {
-            const tmpArray = t.split(', ');
-            finalArray.push(tmpArray[0]);
-            finalArray.push(tmpArray[1]);
-          } else {
-            finalArray.push(t);
-          }
-        } else {
-          finalArray.push('');
-        }
-      });
-      array = finalArray;
-    }
+  commaFlattenArray(array: string[]): string[] {
+    const finalArray: string[] = [];
+    array.forEach((t) => {
+      const i = t.indexOf(', ');
+      if (i > -1) {
+        const tmpArray = t.split(', ');
+        tmpArray.forEach((val) => finalArray.push(val));
```

Comment:
Per discussion with Jess, include all values from each comma separated string, rather than only the first two values. See Input Data column -- 

develop: 
<img width="1500" alt="AG-1328_inputData_develop" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/d61be7a1-00a3-4a1a-a1dc-6b9b4f2b3788">

feature: 
<img width="1494" alt="AG-1328_inputData_feature" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/43ec3a8a-14f4-4f12-8e5d-52b291bcefff">